### DISCUSSION TITLE
Error in login in vercel

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#554] [Help]
I am using auth0 + web3auth.
I followed all the steps required like creating project in web3auth and auth0, creating verifier etc and tested in the localhost with testnet. Everything is working fine.
But after deploying it to the vercel, with the same credentials I used in the localhost, I am getting the following error: 
WalletLoginError: Failed to connect with walletWallet adapter is not ready yet

I have added my deployed vercel url in "Allowed web origins" in the auth0 settings as well

What changed apart from this do I need to make?

**Comment by shahbaz17** [comment]
Hey @KetulCodiste 

Have you added the deployed vercel url in the `Whitelisted URLs` section of the `Plug and Play Project` on Web3Auth Dashboard?

Please add it and try again.

**Reply by KetulCodiste** [reply]
Thank you. It is working now
